package.json

```diff
@@ -28,6 +28,8 @@
     "express": "^4.17.1",
     "nodemon": "^2.0.3",
     "pg": "^8.0.2",
+    "react-calendar": "^3.0.1",
```

Comment:
Whi install these packeages in server side?

Reply:
I think it's a mistake, I delete it